src/cartridges/int_adyen_overlay/cartridge/templates/default/adyenform.isml

```diff
@@ -1,24 +1,23 @@
 <!--- TEMPLATENAME: 3dform.isml --->
-<isdecorate template="checkout/pt_checkout">
-<iscontent type="text/html" charset="UTF-8" compact="true">
-<isinclude template="util/modules"/>
-
-<isinclude template="util/reporting/ReportCheckout.isml"/>
 <div class="checkout">
 	<div class="payerauth">
-		<form action="${pdict.issuerUrl}" method="post" id="3dform" name="3dform">
+		<form method="post" id="3dform" name="3dform">
 			<input type="hidden" name="TermUrl" value="${URLUtils.httpsContinue()}">
 			<input type="hidden" name="PaReq" value="${pdict.paRequest}">
 			<input type="hidden" name="MD" value="${pdict.md}">
 		</form>
 		<script language="javascript">
-			var loaded = false;
+			document.getElementById('3dform').action = sanitizeHTML('${pdict.issuerUrl}');
 			function load(){
 				document.getElementById('3dform').submit();
-				loaded = true;
 			}
 			load();
+
+			function sanitizeHTML(text) {
+				var element = document.createElement('div');
+				element.innerText = text;
+				return element.innerHTML;
+			}
```

Comment:
can't we use jquery for this ?

Reply:
We're trying to reduce jQuery usage in our codebase so eventually we don't use it at all.
the outcome would be the same with jQuery right?

Reply:
We try to avoid using jQuery where possible. We have now most in vanilla JavaScript.